downloads/github-git-cheat-sheet.md

```diff
@@ -64,6 +64,10 @@ When starting out with a new repository, you only need to do it once; either loc
 
 ```$ git init```
 
+After using the init command, the local repository must be linked to the GitHub repository created on our account using the following command:
+
+```$ git remote add origin "https://www.github.com/<<username>>/<<Repository name>>.git"```
```

Comment:
```suggestion
```$ git remote add origin [url]```
```

This slight change would keep consistent syntax between this command and the `git clone` command.

Reply:
-` '$ git origin  "https://www.github.com/<<Salvador1313><Salvador1313> /<Salvador1313>><<https://github.com/brianamarie><https://github.com/brianamarie>.git<https://github.com/brianamarie>>"``